### PR TITLE
coalaOnline: Add support for optional settings

### PR DESCRIPTION
**Pull request by hemangsk**
screen shot 2017-08-08 at 5 26 18 am
screen shot 2017-08-08 at 5 27 34 am

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by sils** [comment]
right align the setting names please

**Comment by hemangsk** [comment]
Screenshot with Right Alignment 
screen shot 2017-08-13 at 9 27 41 am

**Comment by hemangsk** [comment]
Improved Section Card
screen shot 2017-08-13 at 9 40 05 pm


Ability to enter optional settings
screen shot 2017-08-13 at 9 40 20 pm

File: resources/js/app.js [hunk 1]
```diff
@@ -102,3 +102,48 @@ app.filter('orderEmpty', function () {
...
+            match_by_lang = 1;
+          }
+        }
+        if (match == 1) {
+          returnArray.push(bears[x]);
+        }
+      }
+      if (match_by_lang == 1) {
+        for (var x = 0; x < bears.length; x++) {
+          var found = returnArray.some(function (el) {
```
Comment (sils) [inline resources/js/app.js]: this seems overly complicated, why don't you just return all bears that have the language listed plus all that have the all listed and just ignore the returnArray?

**Comment by sils** [comment]
ack e5b8505ed9d3a2f726461c22f7e0c01eda599820 eb1d5635e430bba841c58758d98008beea89d249  6d09ae97447a36029a0f8ed479f6d91c7331fd51

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
@gitmate-bot rebase

**Comment by sils** [comment]
@gitmate-bot would you mind? A rebase please.

**Comment by sils** [comment]
@gitmate-bot would you mind? A rebase please.

**Comment by hemangsk** [comment]
reack a48a08a faf3e0e 450332e

**Comment by sils** [comment]
@rultor merge

**Comment by hemangsk** [comment]
@rultor merge

**Pull request merged**